### PR TITLE
Update SG-09-25-20.md

### PR DESCRIPTION
**Pull request by fgmccabe**

### CODE REVIEW

File: stack/2020/SG-09-25-20.md [hunk 1]
```diff
@@ -6,16 +6,17 @@
...
-The meeting will be on a zoom.us video conference.
-Installation is required, see the calendar invite.
+Francis McCabe
+Zalim Bashorov
+Alon Zakai
+Daniel Hillerström
+Emanuel Ziegler
+Jakob Kummerow
+Ryan Hunt
+Sam Lindley
```
Comment (ngzhian) [inline stack/2020/SG-09-25-20.md:19]: This list looks incomplete, TL, PS, HA, etc are not in the list of participants here.

Comment (lars-t-hansen) [reply stack/2020/SG-09-25-20.md:19]: I was there too.

Comment (penzn) [reply stack/2020/SG-09-25-20.md:19]: Yep, I was there as well.

Comment (fgmccabe) [reply stack/2020/SG-09-25-20.md:19]: Apologies for missing folk. Will try to add.

Comment (KronicDeth) [inline stack/2020/SG-09-25-20.md:24]:
```suggestion
Sam Lindley
Luke Imhoff
```

**Pull request merged**